content/docs/command-reference/run.md

```diff
@@ -215,8 +215,7 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   This is useful if, for example, you need to build a pipeline quickly first,
   and run it all at once later.
 
-- `-f`, `--force` - overwrite an existing `dvc.yaml` file (or the file name
-  determined by the logic described in the `--file` option) without asking for
+- `-f`, `--force` - overwrite an existing `dvc.yaml` file without asking for
```

Comment:
@jorgeorpinel @efiop it should be be probably overwrite an existing stage in the `dvc.yaml` ... otherwise sounds a a bit scary :)

Reply:
oops, yep. Will update